src/commcare_cloud/fab/utils.py

```diff
@@ -173,9 +173,9 @@ def _get_github_credentials(message, force=False):
             "    $ cp {project_root}/config.example.py {project_root}/config.py\n"
             "Then edit {project_root}/config.py"
         ).format(project_root=PROJECT_ROOT))
-        username = input('Github username or token (leave blank to skip): ') or None
-        password = getpass('Github password: ') if username else None
-        GITHUB_CREDENTIALS = (username, password)
+        username = input('Github username (leave blank to skip): ') or None
+        token = getpass('Github token: ') if username else None
+        GITHUB_CREDENTIALS = (username, token)
```

Comment:
Username should not be needed. Does that sound right?
```suggestion
        token = input('Github token (leave blank to skip): ') or None
        GITHUB_CREDENTIALS = (token, None)
```

Reply:
I think you're right and meant to test this myself. I've always put in my username, probably unnecessarily so. Just confirming you typically skip that field and just put in the token?

Reply:
No, if you skip the field it will not prompt for the token, so you currently have to enter both (unnecessarily).